Library/ViewModels/CommentRepliesViewModelTests.swift

```diff
@@ -28,8 +29,14 @@ internal final class CommentRepliesViewModelTests: TestCase {
     self.vm.outputs.loadCommentIntoDataSource.observe(self.loadCommentIntoDataSourceComment.observer)
     self.vm.outputs.loadRepliesAndProjectIntoDataSource.map(second)
       .observe(self.loadRepliesAndProjectIntoDataSourceProject.observer)
-    self.vm.outputs.loadRepliesAndProjectIntoDataSource.map(first)
+    self.vm.outputs.loadRepliesAndProjectIntoDataSource
+      .map(first)
+      .map { $0.0 }
       .observe(self.loadRepliesAndProjectIntoDataSourceReplies.observer)
+    self.vm.outputs.loadRepliesAndProjectIntoDataSource
+      .map(first)
+      .map { $0.1 }
+      .observe(self.loadRepliesAndProjectIntoDataSourceTotalCount.observer)
```

Comment:
Small thing, but could replace the `$0` with `repliesAndProject` to improve readability.